src/test-e2e/e2e.types.ts

```diff
@@ -1,11 +1,13 @@
 export interface E2ECred {
   username: string;
   password: string;
+  guid?: string;
```

Comment:
Can we store this somewhere else? We're cacheing a user guid - its not part of the credential.